src/Features/Core/Portable/CodeRefactorings/UseNamedArguments/AbstractUseNamedArgumentsCodeRefactoringProvider.cs

```diff
@@ -34,7 +34,13 @@ protected abstract class Analyzer<TBaseArgumentSyntax, TArgumentSyntax, TArgumen
                     return;
                 }
 
-                var argument = root.FindNode(context.Span).FirstAncestorOrSelf<TBaseArgumentSyntax>() as TArgumentSyntax;
+                var token = root.FindToken(context.Span.Start);
+                if (token.ValueText == ")" || token.ValueText == ",")
```

Comment:
💭 Personally I would prefer to use the token's kind instead of text (which may require an abstract method implemented by the derived types), but if @CyrusNajmabadi doesn't have a problem with the current approach then I won't block on this.